client/src/includes/initSkipLink.test.js

```diff
@@ -0,0 +1,26 @@
+import initSkipLink from './initSkipLink';
```

Comment:
We should use named exports - unfortunately this is not documented anywhere yet - I will fix this up.